components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -186,9 +186,23 @@ func Run(options ...RunOption) {
 		return
 	}
 
+	endpoint, host, err := cfg.GitpodAPIEndpoint()
+	if err != nil {
+		log.WithError(err).Fatal("cannot find Gitpod API endpoint")
+	}
+
+	experimentsClientOpts := []experiments.ClientOpt{}
+	if cfg.ConfigcatEnabled {
+		experimentsClientOpts = append(experimentsClientOpts, experiments.WithGitpodProxy(host))
+	}
+	exps := experiments.NewClient(experimentsClientOpts...)
```

Comment:
there should be already exps defined above

Reply:
https://github.com/gitpod-io/gitpod/blob/1aa8db11790817494268b6901c84b3636744503b/components/supervisor/pkg/supervisor/supervisor.go#L258C2-L258C6

Reply:
yes, I moved that up